core/src/navigation/ContextSwitcher.html

```diff
@@ -16,15 +16,10 @@
         aria-disabled="{!renderAsDropdown}"
         data-testid="luigi-contextswitcher-button"
       >
-      {#if selectedOption && customSelectedOptionRenderer }
-        {@html customSelectedOptionRenderer(selectedOption)}
-        {:else}
-            {#if !selectedLabel}
-                {$getTranslation(config.defaultLabel)}
-            {:else}
-                {selectedLabel}
-            {/if}
-          <i class="sap-icon--megamenu fd-shellbar__button--icon"></i>
+        {#if selectedOption && customSelectedOptionRenderer } {@html
+        customSelectedOptionRenderer(selectedOption)} {:else} {#if !selectedLabel}
+        {$getTranslation(config.defaultLabel)} {:else} {selectedLabel} {/if}
+        <i class="sap-icon--megamenu fd-shellbar__button--icon"></i>
```

Comment:
Lint issue?  The previous structure is more readable. Please change it back.

Reply:
Lint change it like this.